src/App.js

```diff
@@ -1,35 +1,38 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import { BrowserRouter, Route } from "react-router-dom";
+import "./App.css";
+import Header from "./components/Header";
+import Home from "./components/Home";
+import Project from "./components/Project";
+import WorksList from "./components/WorksList";
+import data from "./data.json";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
+    let project;
+    data.forEach((proj) => {
+        return (project = proj);
+    });
+    let { id, year, title, img, technologies } = project;
+    return (
+        <BrowserRouter>
+            <div className="App">
+                <header className="App-header">
+                    <Header />
+                </header>
+                <main className="App-main">
+                    <Route exact path="/" component={Home} />
+                    <Route exact path="/" component={WorksList} />
 
-      </header>
-      <main>
-        <div className='profile'>
-
-        <div className='profile-title'>
-          <h1>Greetings, i am Mohamad.</h1>
-          <p>A Web Developer</p>
-          <p> from Berlin.</p>
-        </div>
-        <div className='profile-description'>
-          <p>
-          I am a full stack web developer with a strong passion for coding , I started as a self-taught , and after a couple of online courses and weekend workshops I fully committed to learning and joined an on-site intensive bootcamp to get more familiar with the cutting edge technologies and industry standards. I am eager to be a part of a motivated team to bring fresh ideas and grow together.
-          </p>
-          <p> <strong>Frontend Tech :</strong>  React, Vue, NextJs, jQuery, Bootstrap, Material Design, Bulma, CSS / SCSS Styled Components
-</p>
-<p> <strong> Backend Tech :</strong>  Node.js, PHP, Express, MongoDB, SQL</p>
-        </div>
-        </div>
-      </main>
-      <footer></footer>
-
-    
-    </div>
-  );
+                    <Route
+                        exact
+                        path={`/project/${title}`}
+                        render={() => <Project proj={project} />}
+                    />
```

Comment:
use this new syntax it is much cleaner : 
```javascript 
   <Route path="/your-path">
      <YourComponent yourProps... />
    </Route>
```

but as I can tell here ..you want to pass the list of projects to your project component .. then you should build a Projects component that you pass all the projects to and then map them inside it to the Project component

Reply:
i've already used this now
` <Route path={`/project/:title`} component={Project} />`

Reply:
this works but you don't have a way to pass props to your Project component